team-best-practices.md

```diff
@@ -0,0 +1,191 @@
+
+# Team best practices
+
+This document aims to address contibutors best practices of the following repositories:
+ - remix-ide https://github.com/ethereum/remix-ide
+ - remix https://github.com/ethereum/remix
+ - remixd https://github.com/ethereum/remixd
+
+This document is not in its final version, **a team meeting which aim to address new/old best practices, feedback, workflows, all kind of issues related to how the team work together occurs every 2 weeks.**
+This document link to other specialised best practices (like coding best practices).
+
+Related links:
+ - Public WebSite: https://remix-project.org
+ - Remix basic FAQ: https://hackmd.io/KVooMJhWRImCGq6zkDgW9A
+ - Remix live: https://remix.ethereum.org
+ - Remix alpha live: https://remix-alpha.ethereum.org
+ - Remix beta live: https://remix-beta.ethereum.org
+ - Remix-ide NPM module: https://www.npmjs.com/package/remix-ide
+ - Remix-lib NPM module: https://www.npmjs.com/package/remix-lib
+ - Remix-tests NPM module: https://www.npmjs.com/package/remix-test
```

Comment:
libs link will be updated with the prefix: https://www.npmjs.com/package/@remix-project